README.md

```diff
@@ -41,6 +63,13 @@ if ((n - Math.floor(n)) >= .5) {
 }
```

Comment:
2a) It's possible to go even simpler! Remember that Math has lots of other functions available; you're not bound to use only the ones used in the first iteration of the problem.